index.css

```diff
@@ -0,0 +1,346 @@
+* {
+  font-family: "Raleway", sans-serif;
+  /* background-color: black; */
+}
+.container {
+  background-color: blue;
+  color: white;
+  aspect-ratio: 16/9;
+}
+/* .imgContainer {
```

Comment:
please remove unnecessary code if you want please comment a proper message

Reply:
done